src/components/TextField.vue

```diff
@@ -94,6 +97,19 @@ export default {
     multiline: {
       type: Boolean,
     },
+    /**
+    * Display currency text field
+    */
+    currency: {
+      type: Boolean,
+    },
```

Comment:
```suggestion
    currency: Boolean,
```

Reply:
updated for required and multiline as well.